src/entities/Game.ts

```diff
@@ -0,0 +1,28 @@
+import Player from "./Player";
+
+class Game {
+    tileDeck;
+    tileDiscard;
+    eventDeck;
+    eventDiscard;
+    players;
+    eventCardIsActive: boolean;
+    eventCardTurns: number;
+
+    constructor(numberPlayers) {
+
+        //Event cards never active at game start
+        this.eventCardIsActive = false;
+        this.eventCardTurns = 0;
+        
+        // Add all players
+        var i;
+        this.players = Array(numberPlayers)
+        for (i = 0; i < numberPlayers; i++) {
```

Comment:
tip: You can define your counter inside your loop

```
        for (let i = 0; i < numberPlayers; i++) {
        }
```

Value of this being that `i` is now only in the scope of that for loop. In your code, it's scope is the entire function.
```suggestion
        for (let i = 0; i < numberPlayers; i++) {
```